packages/lib/services/synchronizer/Synchronizer.conflicts.test.ts

```diff
@@ -47,6 +47,10 @@ describe('Synchronizer.conflicts', function() {
 		for (const n in conflictedNote) {
 			if (!conflictedNote.hasOwnProperty(n)) continue;
 			if (n == 'id' || n == 'is_conflict') continue;
+			if (n == 'conflict_parent_id') {
+				expect(conflictedNote[n]).toBe(note2conf.id);
+				continue;
+			}
```

Comment:
Makes no sense to have this here. Move it out of the loop please

Reply:
`conflict_parent_id` is kinda a special property since its value is not the same on both the conflict note and the original note.. If I don't handle it specially, the test will fail in the below line: `expect(conflictedNote[n]).toBe(note2conf[n]);`

Reply:
`if (n == 'id' || n == 'is_conflict' || n == 'conflict_parent_id') continue;`

then above:

`expect(conflictedNote. conflict_parent_id).toBe(note2conf.id);`